aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -553,27 +553,28 @@ AT_ERROR("solve: MAGMA library not found in "
 
     MAGMAQueue magma_queue(b.get_device());
 
+    constexpr int64_t batch_limit = 65535;
```

Comment:
This might be better as a global constant so we don't have to repeat it three times?

Reply:
#24438 defines another `batch_limit` for `cholesky`, so I thought that we could keep them separate (one for each function).

Reply:
Oh I see. That's reasonable, thanks for the explanation